docs/tutorials/depth/videos_monodepth2.py

```diff
@@ -0,0 +1,196 @@
+    t = time.time()
+    pred_disp = pred_disp.squeeze().as_in_context(mx.cpu()).asnumpy()
+    pred_disp = cv2.resize(src=pred_disp, dsize=(original_width, original_height))
+    pred_disp_sequences.append(pred_disp)
+
+    pred_depth = 1 / pred_disp
+    pred_depth *= scale_factor
+    pred_depth[pred_depth < MIN_DEPTH] = MIN_DEPTH
+    pred_depth[pred_depth > MAX_DEPTH] = MAX_DEPTH
+    pred_depth_sequences.append(pred_depth)
+
+
+##############################################################################
+# Store results
+# -----------------
+#
+# Here, we provide an example of storing the prediction results. Including:
+#
+# - store depth map
+output_path = os.path.join(os.path.expanduser("."), "tmp")
+
+pred_path = os.path.join(output_path, 'pred')
+if not os.path.exists(pred_path):
+    os.makedirs(pred_path)
+
+for pred, file in zip(pred_depth_sequences, files):
+    pred_out_file = os.path.join(pred_path, file)
+    cv2.imwrite(pred_out_file, pred)
+
+##############################################################################
+# - store disparity and save it to a video
+rgb_path = os.path.join(output_path, 'rgb')
+if not os.path.exists(rgb_path):
+    os.makedirs(rgb_path)
+
+output_sequences = []
+for raw_img, pred, file in zip(raw_img_sequences, pred_disp_sequences, files):
+    vmax = np.percentile(pred, 95)
+    normalizer = mpl.colors.Normalize(vmin=pred.min(), vmax=vmax)
+    mapper = cm.ScalarMappable(norm=normalizer, cmap='magma')
+    colormapped_im = (mapper.to_rgba(pred)[:, :, :3] * 255).astype(np.uint8)
+    im = pil.fromarray(colormapped_im)
+
+    raw_img = np.array(raw_img)
+    pred = np.array(im)
+    output = np.concatenate((raw_img, pred), axis=0)
+    output_sequences.append(output)
+
+    pred_out_file = os.path.join(rgb_path, file)
+    cv2.imwrite(pred_out_file, cv2.cvtColor(pred, cv2.COLOR_RGB2BGR))
+
+width = int(output_sequences[0].shape[1] + 0.5)
+height = int(output_sequences[0].shape[0] + 0.5)
+fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+out = cv2.VideoWriter(
+    os.path.join(output_path, 'demo.mp4'), fourcc, 20.0, (width, height))
+
+for frame in output_sequences:
+    frame = cv2.cvtColor(frame, cv2.COLOR_RGB2BGR)
+
+    out.write(frame)
+    cv2.imshow('demo', frame)
+
+    if cv2.waitKey(25) & 0xFF == ord('q'):
+        break
+
+##############################################################################
+# We release the webcam before exiting:
+out.release()
+cv2.destroyAllWindows()
+
+
+##############################################################################
+# You can start with the example code.
+# -----------------
+#
+# Download the script to run the demo
+#
+# :download:`Download cam_demo.py<../../../scripts/depth/demo.py>`
+#
+# Example running command::
+#
+#         python demo.py --model_zoo monodepth2_resnet18_kitti_mono_stereo_640x192 --input_format image --data_path ~/.mxnet/datasets/kitti/example/2011_09_26/2011_09_26_drive_0095_sync/image_02/data --output_format video
```

Comment:
This is a command for storing videos. It is better to put another command to storing in image sequences because maybe more people are interested in that. Thank you.

Reply:
I have provided them, thanks for your suggestions!